src/data/F19_tabulated.dat

```diff
@@ -0,0 +1,103 @@
+x      k_3.02  deltak  sigmak
```

Comment:
we are actually already making use of a data-providing package `DataDeps`. If this data is available online already, I suggest we switch to using DataDeps for providing it.

Reply:
Oh yea, I saw that was used for the dust map. For sure!

Reply:
Ok, just tried it out (fef8bdc)

At first I tried sticking with the using constants approach by just directly swapping out the read path from this:
```julia
const data_x, data_k, data_delta_k, data_sigma_k = let data = readdlm(joinpath(dirname(pathof(DustExtinction)), "data", "F19_tabulated.dat"), skipstart=1)
    (data[:, i] for i in 1:4)
end
```

to this:
```julia
const data_x, data_k, data_delta_k, data_sigma_k = let data = readdlm(joinpath(datadep"F19", "F19_tabulated.dat"), skipstart=1)
    (data[:, i] for i in 1:4)
end
```

where `F19` is the data directory I registered in `src/DustExtinction.jl`. This seems to work fine if I just run parts of the package (i.e. importing DataDeps, then running just the lines above), but attempting import the whole package leads to a KeyError when trying to point to the data via `datadep"F19"`. 

For some reason, it looks like the string macro can't be resolved unless it is inside a function scope? I tested this out with the same error for `datadep"sfd98_map"` by just trying to print it right above [where it is called in a function](https://github.com/JuliaAstro/DustExtinction.jl/blob/master/src/dust_maps.jl#L58), so I am pretty confused about why this doesn't work.

For now I just put everything inside the internal `f19_invum` function (fef8bdc), but I don't know how that compares in terms of performance to just using constants.